ddtrace/opentracer/tracer.py

```diff
@@ -337,3 +340,12 @@ def extract(self, format, carrier):  # noqa: A002
         dd_span_ctx = ot_span_ctx._dd_context
         self._dd_tracer.context_provider.activate(dd_span_ctx)
         return ot_span_ctx
+
+    def get_log_correlation_context(self):
+        # type: () -> Optional[DDLogRecord]
+        """Retrieves the Correlation Identifiers for the current active ``Trace``
```

Comment:
```suggestion
        """Retrieves the Correlation Identifiers for the current active trace
```

we don't have a public data structure for traces